source/cloud-security/azure/activity-services/services/storage.rst

```diff
@@ -23,7 +23,7 @@ This section explains how to use the Azure portal to archive the Azure activity
 Configuring the Activity log export
 -----------------------------------
 
-1. To export the logs, search for the **Activity log** service. It can be found by typing "Activity" in the search engine. From there, access the **Audit Logs** section and click on **Export Data Settings**.
+1. To export the logs, search for the **Activity log** service. It can be found in **Azure Active Directory**. From there, access the **Audit Logs** section and click on **Export Data Settings**.
```

Comment:
I couldn't find the **Audit Logs** section following these instructions.

![activity_log](https://github.com/wazuh/wazuh-documentation/assets/51374959/a1671b6a-6ff8-4f87-b123-3b54ca3a388f)

It seems that it is now a differente service.

![audit_logs](https://github.com/wazuh/wazuh-documentation/assets/51374959/331af692-5cc3-43cb-b346-078cc7b8656d)